eng/tools/azure-sdk-tools/packaging_tools/code_report.py

```diff
@@ -188,7 +188,7 @@ def create_report_from_func(function_attr):
 def resolve_package_directory(package_name):
     packages = [
         os.path.dirname(p)
-        for p in (glob.glob("{}/setup.py".format(package_name)) + glob.glob("sdk/*/{}/setup.py".format(package_name)))
+        for p in (glob.glob("{}/pyproject.toml".format(package_name)) + glob.glob("sdk/*/{}/pyproject.toml".format(package_name)))
```

Comment:
There may still be some packages will use `setup.py`, so we shall update logic to be compatible with setup.py.